hazelcast/src/main/java/com/hazelcast/jet/impl/JobSummary.java

```diff
@@ -102,6 +118,7 @@ public String getNameOrId() {
      * @deprecated use {@link #getNameOrId()}, semantics is the same
      */
     @Nonnull
+    @Deprecated
```

Comment:
Why is this deprecated?

Reply:
SonarLint reported missing `@Deprecated` annotation as an issue. You marked this method as deprecated in Java Docs :)